analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -79,14 +79,21 @@ class ClangTidy(analyzer_base.SourceAnalyzer):
     def add_checker_config(self, checker_cfg):
         LOG.error("Not implemented yet")
 
+    @classmethod
+    def invocation_with_all_checks(cfg_handler):
+        return [cfg_handler.analyzer_binary, "-checks=*"]
+
     @classmethod
     def get_analyzer_checkers(cls, cfg_handler, environ):
         """
         Return the list of the supported checkers.
         """
         try:
+            list_invocation =\
+                cls.invocation_with_all_checks(cfg_handler)\
```

Comment:
I think there is no need to create a separate function for this. You can add this option to the original list.

Also could you please extend the documentation of this function a little bit more that this function will return the list of `ALL` the supported checkers?

Reply:
Fixed, and modified accordingly.